docs/CLI/index.md

```diff
@@ -17,30 +17,30 @@ Install from the latest [release artifacts](https://github.com/oras-project/oras
 ### Linux
 
 ```bash
-curl -LO https://github.com/oras-project/oras/releases/download/v0.16.0/oras_0.16.0_linux_amd64.tar.gz
+curl -LO https://github.com/oras-project/oras/releases/download/v1.0.0-rc.1/oras_1.0.0-rc.1_linux_amd64.tar.gz
 mkdir -p oras-install/
-tar -zxf oras_0.16.0_*.tar.gz -C oras-install/
+tar -zxf oras_1.0.0-rc.1_*.tar.gz -C oras-install/
 mv oras-install/oras /usr/local/bin/
-rm -rf oras_0.16.0_*.tar.gz oras-install/
+rm -rf oras_1.0.0-rc.1_*.tar.gz oras-install/
 ```
 
 ### macOS
 
 ```bash
-curl -LO https://github.com/oras-project/oras/releases/download/v0.16.0/oras_0.16.0_darwin_amd64.tar.gz
+curl -LO https://github.com/oras-project/oras/releases/download/v1.0.0-rc.1/oras_1.0.0-rc.1_darwin_amd64.tar.gz
```

Comment:
Nowadays, all new macs are ARM-based where the 1st mac with Apple silicon M1 chip was released on Nov 10 2020 (2 years ago).
```suggestion
curl -LO https://github.com/oras-project/oras/releases/download/v1.0.0-rc.1/oras_1.0.0-rc.1_darwin_arm64.tar.gz
```

Reply:
Something like `ARCH=$(test $(uname -m) = arm64 && echo arm64 || echo amd64)` maybe?

Reply:
@TerryHowe Thanks for your suggestion. IMO, it looks flexible but is not simple and intuitive if using the environmental variable and shell script here. I have changed the steps a little bit and add a note for a different arch machine.